### PR TITLE
answers for lesson 1

### PR DESCRIPTION
**Pull request by IrynaLyakhova**

### CODE REVIEW

File: 01-node-tutorial/answers/11-fs-async.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const { writeFile } = require("fs");
+console.log("at start");
+
+writeFile("./temporary/fileB.txt", "This is line 1\n", (err, result) => {
+  console.log("at point 1");
+  if (err) {
+    console.log("This error happened: ", err);
+  } else {
+    console.log("Done with this task")
```
Comment (ethanarns) [inline 01-node-tutorial/answers/11-fs-async.js:9]: You got the command correct, however, there needs to be more writeFile calls in the callback

File: 01-node-tutorial/answers/12-http.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    res.end(`
+    <h1>Oops!</h1>
+    <p>We can't seem to find the page you are looking for</p>
+    <a href="/">back home</a>
+    `)
+  }
+})
+
+console.log("Listening on port 5000")
+server.listen(5000)
```
Comment (ethanarns) [inline 01-node-tutorial/answers/12-http.js:18]: Make sure to change the port value!

**Review by ethanarns — CHANGES_REQUESTED** [review]
Excellent work! Only a few minor issues were found in the submitted files, good job. However, you seem to be missing prompter.js, could you complete that assignment, and then resubmit this?

**Comment by IrynaLyakhova** [comment]
@ethanarns thanks for the review! I've addressed your comments, please take another look.

**Pull request closed without merging**